component-resolver-core/src/main/groovy/org.octopusden/octopus/escrow/configuration/validation/EscrowConfigValidator.groovy

```diff
@@ -411,6 +412,25 @@ class EscrowConfigValidator {
         }
     }
 
+    def validateBuildConfigurationTools(EscrowModuleConfig moduleConfig) {
+        def tools = moduleConfig.getBuildConfiguration()?.getTools()
+        tools?.each { tool ->
+            def toolName = tool.getName()
+            if (toolName == null)  {
+                registerError("tool is not configured correctly")
```

Comment:
let's change error description to more specific, like "tool name is not specified"